e2e/pages/caseList.page.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     caseState: '#wb-case-state',
     evidenceHandled: '#evidenceHandled-Yes',
     evidenceNotHandled: '#evidenceHandled-No',
+    caseId: 'CCD Case Number',
```

Comment:
The others seem to use IDs instead of the text, would it be better to be consistent?

Reply:
this element has id in a format '[CASE_REFERENCE]' and selector like '#[CASE_REFERENCE]' is not valid and is not recognised by codecept

Reply:
fair enough